tests/test_sqlite_adapter.py

```diff
@@ -0,0 +1,249 @@
+import platform
+import sqlite3
+import sys
+from datetime import datetime, timezone
+
+import pytest
+
+from flow.record import Record, RecordDescriptor, RecordReader, RecordWriter
+from flow.record.adapter.sqlite import sanitized_name
+
+
+def generate_records(amount) -> Record:
```

Comment:
We really ought to make this a pytest fixture at some point.

Reply:
haha yes, totally agree. there is actually some helper functions in `tests/_utils.py`, file should probably be called something else and maybe indeed nicer if it could be an actual pytest fixture.